### ISSUE TITLE
[Bug] TabIndex ignored in Xamarin.Forms for Mac

### ISSUE THREAD

**Issue by JohnHDev** [xamarin/Xamarin.Forms#11458]
### Description
TabIndex ignored in Xamarin.Forms for Mac
### Steps to Reproduce

1. Run the attached example with the Mac set as startup project
2. 3 entries are displayed, press tab to change the currently selected

### Expected Behavior
Order should be entry1, entry3 and then entry2

### Actual Behavior
Order is as expressed, so entry1, entry2 and entry3

### Basic Information

- Version with issue:
- Last known good version: None
- IDE:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

### Reproduction Link

[XF4iOSAndroidMacDemo.zip](https://github.com/xamarin/Xamarin.Forms/files/4933451/XF4iOSAndroidMacDemo.zip)

### Workaround
None

**Comment by rachelkang** [comment]
Hi, @JohnHDev - as elaborated in https://github.com/dotnet/maui/issues/1646 and https://devblogs.microsoft.com/xamarin/the-journey-to-accessible-apps-meaningful-content-ordering/, we no longer advise that TabIndex be used. Unfortunately, the property was long misunderstood and misused.

Ideally, your app should not manipulate content ordering whatsoever - should you really need to, however, we recommend checking out [SemanticOrderView](https://github.com/xamarin/XamarinCommunityToolkit/pull/1240) from the XamarinCommunityToolkit, which has already proven to be a more accessible alternative.

If you haven't already, I also recommend updating to the latest version Xamarin.Forms or at least to Xamarin.Forms 5.0.0.2083, which has many accessibility improvements. These improvements include an experimental accessibility flag that addresses a lot of improvements in Android accessibility.

To leverage this flag on the latest versions of Xamarin Forms, the following code can be inserted in the App.xaml.cs file, following the InitializeComponent call and before the MainPage is set: `Xamarin.Forms.Device.SetFlags(new List<string> { "Accessibility_Experimental"});`

Let us know how that works for you!
